routes/api.js

```diff
@@ -219,4 +219,20 @@ router.post("/setcommand/:detector", checkKey, function(req, res) {
   }).catch((err) => res.json({message: err.message}));
 });
 
+function GetSystemMonitorStatus(collection, host, callback) {
+  var query = {host: host,};
+  var options = {sort: {'_id': -1}, limit: 1};
+  return collection.find(query, options);
+}
+router.get("/getsystemmonitor/:host", checkKey, function(req, res) {
```

Comment:
I think `gethoststatus` would be a better url, as this matches more with the other `getstatus` url, which is doing something very similar. In principle we could reuse the helper function for both, because there's nothing system-monitor specific about `GetSystemMonitorStatus`.

Reply:
Changed the name. We could reuse the helper function but then we'll need to pass a query to the function since they are different

Reply:
The query is effectively the same, the only difference being a) which collection, and b) the hostname. Something we probably should think about is expanding the (undocumented) support for more than just the most recent document.